auth_from_http_remote_user/controllers/main.py

```diff
@@ -0,0 +1,83 @@
+# Author: Laurent Mignon
+# Copyright 2014-2018 'ACSONE SA/NV'
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+import logging
+import werkzeug
+
+from odoo import http
+from odoo import api
+from odoo import SUPERUSER_ID
+from odoo.http import request
+from odoo.addons.web.controllers import main
+
+from .. import utils
+
+_logger = logging.getLogger(__name__)
+
+
+class Home(main.Home):
+
+    _REMOTE_USER_ATTRIBUTE = 'HTTP_REMOTE_USER'
+
+    @http.route('/web', type='http', auth="none")
+    def web_client(self, s_action=None, **kw):
+        main.ensure_db()
+        try:
+            self._bind_http_remote_user(http.request.session.db)
+        except http.AuthenticationError:
+            return werkzeug.exceptions.Unauthorized().get_response()
+        return super(Home, self).web_client(s_action, **kw)
+
+    def search_user(self, users, login):
+        """Search for an active user by login name"""
+        user = users.sudo().search([
+            ('login', '=', login),
+            ('active', '=', True)],
+            limit=1
+        )
+        if user:
+            return user[0]
+        return None
+
+    def logging_http_remote_user(self, env, user):
```

Comment:
@TDu @guewen I think this meth is never used since it has been renamed https://github.com/OCA/server-auth/pull/7/files#diff-00f6cb1d0bf2f276bf85d1ccf7615dfbR43